src/Native/CMakeLists.txt

```diff
@@ -62,6 +65,8 @@ if(WIN32)
     list(APPEND RESOURCES ${CMAKE_SOURCE_DIR}/../../Tools/NativeVersion.rc)
 else()
     add_compile_options(-Wno-unused-local-typedef)
+    add_compile_options(-fPIC)
```

Comment:
I originally missed these setting because I copied [the "common" CMake file from dotnet/core-setup](https://github.com/dotnet/core-setup/blob/04035b3a4c4764f57537045428989a09682631f4/src/settings.cmake#L152). However, `/MT` for Windows and these two settings on Unix aren't in the common CMake file in that repo, but are spread to the individual CMake files for each library:

https://github.com/dotnet/core-setup/blob/8fe49b4211abaa649784966a4f6b55a38a237d9e/src/corehost/cli/hostpolicy/CMakeLists.txt#L7-L14
https://github.com/dotnet/core-setup/blob/8fe49b4211abaa649784966a4f6b55a38a237d9e/src/corehost/cli/fxr/CMakeLists.txt#L7-L14

Which is why they were overlooked originally.